gripper/firmware/utility_gpio.c

```diff
@@ -3,6 +3,18 @@
 #include "platform_specific_hal_conf.h"
 #include "stm32g4xx_hal_gpio.h"
 
+#if PCBA_PRIMARY_REVISION == 'c'
```

Comment:
this feels like it should be a different PR since it's doing something pretty unrelated

Reply:
Oh I should've put it in the description; we had to switch these GPIOs out in order to add the new encoder. So I'm gonna try to keep these here, otherwise, the encoder wouldn't work properly.